plugins/integration-openai/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { VariableDictionary } from "@amplication/code-gen-types";
+
+export const envVariables: VariableDictionary = [
+  { OPENAI_API_KEY: "[open-ai-key]" },
+];
```

Comment:
This being a secret might be better to not expose it by environment variable. 
Not sure we have any abstraction for a secret manager, have we @yuval-hazaz ?

Reply:
I agree
I will look into it

Reply:
To keep things simple for the initial phase and not complicate the process, I added a note to the README file of the plugin
for the next phase, we can think about how plugins can ask the user to add secrets or how we use secret manager integrally

Reply:
I think it's good enough for now with the comment. We do not handle deployment at the moment and I feel it's fine to leave this to the user to handle per his actual deployment environment (is he using Vault? AWS? Azure? 1Password? - we can't know, so the .env file as a dev-time solution is okay IMO).